### PR TITLE
Adds in App Instance Property support

### PR DESCRIPTION
**Pull request by IanHakes-okta**
## Description:
- **What's changed?** 
Updates descriptions for all three protocols (SAML, SCIM, OIDC) with instructions on how to configure variables and use them in ISV scenarios that use multi-tenants environments.

- **Is this PR related to a Monolith release?**
Yes. 2020.11.0

### Resolves:

* [OKTA-328618.](https://oktainc.atlassian.net/browse/OKTA-328618.)

### CODE REVIEW

**Review by azhariqbal-okta — APPROVED** [review]
🚀

File: packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/saml2/submit.md [hunk 1]
```diff
@@ -1,24 +1,44 @@
...
+  * **Label Name** &mdash; A descriptive name for the dynamic variable that administrators see when installing your app integration.
+  * **Variable Name** &mdash; an automatically generated variable used when constructing the dynamic address. This is hidden from admins and is only passed to your external application.
+  * **Help Text** &mdash; Any descriptive text to be shown to administrators when installing your app integration.
+  * **Type** &mdash; The property type for your parameter. Options are "String", "URL", or "HTTPS URL".
+  * Click **Save** to add the variable to the list.
+
+    After the variable is created, you can click the pencil icon to make changes to the details, the clipboard icon to copy the **Variable Name** to your local clipboard, or the "X" icon to remove the variable entirely.
+
+  * **Construct your dynamic ACS URL by copying the variables above and pasting them where applicable** &mdash; Provide your complete Assertion Consumer Service (ACS) URL endpoint where Okta will post SAML responses for your app integration.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/saml2/submit.md:29]:
```suggestion
  * **Construct your dynamic ACS URL by copying the variables above and pasting them where applicable** &mdash; Provide your complete Assertion Consumer Service (ACS) URL endpoint where Okta posts SAML responses for your app integration.
```

File: packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/scim/submit.md [hunk 2]
```diff
@@ -1,11 +1,26 @@
...
 
-* **Base URL (enter “n/a” if the URL varies by tenant)** &mdash; Provide the base URL that your integration uses when sending an outbound call to your SCIM server. This should be a valid production endpoint. If the the URL varies according to which tenant is using it (e.g. `lasvegas.example.com/scim/v2/` or `seattle.example.com/scim/v2`), then type `n/a`. Explain in your configuration guide how your users should construct the URL while configuring the settings for your application.
+* **Does your Base URL vary per tenant?** &mdash; If **Yes**, a new field appears to assist you in setting up a per tenant configuration.
+  * **What variables do your admins need to specify to install your app?** &mdash; When you click **Add Variable**, the interface displays a dialog box to collect the following information:
+  * **Label Name** &mdash; A descriptive name for the dynamic variable that administrators see when installing your app integration.
+  * **Variable Name** &mdash; an automatically generated variable used when constructing the dynamic address. This is hidden from admins and is only passed to your external application.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/scim/submit.md:11]:
```suggestion
  * **Variable Name** &mdash; An automatically generated variable used when constructing the dynamic address. This is hidden from admins and is only passed to your external application.
```

**Review by susanharper-okta — APPROVED** [review]
one comment. looks clean built locally, also. great job.

File: packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/scim/submit.md (hunk 2 above)
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/scim/submit.md]: Fixed in all three files

**Pull request merged**